JDI.Light/JDI.Light.Tests/UIObjects/TestSite.cs

```diff
@@ -30,12 +30,12 @@ public class TestSite : WebSite
         [Page(Url = "simple-table.html", Title = "Simple Table")]
         public SimpleTablePage SimpleTablePage;
 
+        [Page(Url = "/html5.html", Title = "HTML 5")]
+        public Html5Page Html5Page;
```

Comment:
this can be a property instead of a field

Reply:
Other members are fields. Hwy do we need prop in this case? What should me written in setter/getter?